vipps-qr-api.md

```diff
@@ -171,7 +171,7 @@ Body:
 }
 ```
 
-The response will be similar to this, where the URL in the responseBody will be a link to the image as defined in the accept header:
+The response will be similar to this, where the URL in the `responseBody` will be a link to the image as defined in the accept header:
```

Comment:
maybe "response body" or "body of the response"?

Reply:
Changed to   "body of the response"